src/ray/gcs/gcs_server/error_info_handler_impl.cc

```diff
@@ -0,0 +1,31 @@
+#include "error_info_handler_impl.h"
+
+namespace ray {
+namespace rpc {
+
+void DefaultErrorInfoHandler::HandleReportJobError(
+    const ReportJobErrorRequest &request, ReportJobErrorReply *reply,
+    SendReplyCallback send_reply_callback) {
+  JobID job_id = JobID::FromBinary(request.error_data().job_id());
+  RAY_LOG(DEBUG) << "Reporting job error, job id = " << job_id
+                 << ", type = " << request.error_data().type();
+  auto error_table_data = std::make_shared<ErrorTableData>();
+  error_table_data->CopyFrom(request.error_data());
+  auto on_done = [job_id, request, send_reply_callback](Status status) {
```

Comment:
It is better to capture `type` instead of the whole `request`.

Reply:
fixed